modules/test/playwright/tests/object-web/objectLayout.spec.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {editObjectDefinitionPagesTest} from '../../fixtures/editObjectDefinitionPagesTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {objectPagesTest} from '../../fixtures/objectPagesTest';
+import getRandomString from '../../utils/getRandomString';
+import getRandomObjectField from './utils/getRandomObjectField';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	editObjectDefinitionPagesTest,
+	featureFlagsTest({
+		'LPS-187854': true,
+	}),
+	loginTest(),
+	objectPagesTest
+);
+
+test.describe('Manage Object Layouts through the Object Layout tab', () => {
+	test('Can view all fields of an object when creating its layout', async ({
+		apiHelpers,
+		editObjectDefinitionPage,
+		page,
+		viewObjectDefinitionsPage,
+		viewObjectLayoutsPage,
+	}) => {
+		const fields = getRandomObjectField({fieldsQuantity: 20});
+
+		const objectDefinition =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition({
+				fields,
+				status: {code: 0},
+			});
+
+		const objectFields = objectDefinition.objectFields;
+
+		const objectFieldLabelList = [];
+
+		const layoutName = getRandomString();
+
+		objectFields.forEach((objectField) => {
+			objectFieldLabelList.push(objectField.label.en_US);
+		});
+
+		await viewObjectDefinitionsPage.goto();
+
+		await viewObjectDefinitionsPage.clickEditObjectDefinitionLink(
+			objectDefinition.name
+		);
+
+		await editObjectDefinitionPage.openLayoutsTab();
```

Comment:
Hey @vngoliveira, we can create a goto function in `ObjectLayoutsPage`. Check the `ObjectValidationsPage` goto function.